src/Api/fetchCitiesToSearcher.tsx

```diff
@@ -0,0 +1,21 @@
+const fetchDataCity = async (url) => {
```

Comment:
Let's try to avoid adding `Data` everywhere, to some extent everything is a piece of data. `fetchCity` is enough 😃